ofrak_core/ofrak/model/component_model.py

```diff
@@ -18,7 +18,7 @@ class ComponentConfig:
     """
 
 
-@dataclass(frozen=True)
+@dataclass(unsafe_hash=True)
```

Comment:
I'd rather not change this.

The `ComponentExternalTool` is a nice, simple, stateless class. Can you cache the tool installation state in the `GzipUnpacker`? Until we have more use cases for caching this result I'd rather keep this class simple and immutable.